frontend/packages/ux-editor/src/components/config/editModal/EditGrid/types/ViewSize.ts

```diff
@@ -0,0 +1,7 @@
+export enum ViewSize {
+  xs = 'xs',
+  sm = 'sm',
+  md = 'md',
+  lg = 'lg',
+  xl = 'xl',
+}
```

Comment:
While reading the [documentation on enums](https://www.typescriptlang.org/docs/handbook/enums.html), it seems preferable to adhere to PascalCase for enum values.

```suggestion
export enum ViewSize {
  Xs = 'xs',
  Sm = 'sm',
  Md = 'md',
  Lg = 'lg',
  Xl = 'xl',
}
```